### ISSUE TITLE
Select2 as checkboxes instead of a list of items?

### ISSUE THREAD

**Issue by crazyserb** [Mottie/tablesorter#1007]
Is there a way to format select2 filter field so it shows up as a popup with checkboxes of some sort, without affecting the width/height of the field itself, as it does in this default example:

http://prntscr.com/8c2biw

As in, the more items I select in there the more that field gets stretched and pushed down in height. I'd rather have it stay the same height/width and have some sort of a popup or something with possibly checkboxes or... something cleaner, I dunno.

Any ideas?

**Comment by Mottie** [comment]
Hi @crazyserb!

That sounds like a question to ask someone who takes care of the select2 plugin.

I don't think there is an option, but I do remember a different select plugin that displays something like "and XX other options" when there are multiple choices. I don't remember its name.

**Comment by Mottie** [comment]
Ahh, ok I found it: http://davidstutz.github.io/bootstrap-multiselect/

If you select more than `numberDisplayed` options, it will display "# selected" instead of a list of all the options.

Right now, you'll need to write custom `filter_formatter` code to incorporate that plugin. But, I started work on a new `filter_customVal` option which will essentially replace the `filter_formatter` function by allowing you to write functions to get & set filter row component values and use a completely custom filter row... I just haven't had the time nor motivation to debug it lately.

**Comment by crazyserb** [comment]
Yeah, writing a custom filter_formatter is way above my head at the
moment... so if you ever pull that off yourself, for this type of
functionality, let me/us know so we can update the code on our end. Or if
you are open to that, can I hire you/pay you to do this for me, for the
time being?
